app/assets/javascripts/components/markdown-editor.js

```diff
@@ -33,16 +36,26 @@ MarkdownEditor.prototype.handlePreviewButton = function (event) {
   // Mirror textarea's height
   $preview.style.height = this.$input.offsetHeight + 'px'
 
-  // Render markdown
-  window.marked(text, function (err, content) {
-    if (err) {
-      $preview.innerHTML = 'Error previewing content'
-      throw err
-    } else {
-      $preview.innerHTML = content
-    }
-  })
-
+  if (text) {
+    // Render markdown
+    window.marked(
+      text,
+      {
+        gfm: true,
```

Comment:
I'm not really sure what this one's doing. I tried 'hello<newline>there' and 'hello<space><space><newline>there' and in the latter the editor inserted a '<dot>' in edit mode. Perhaps this bit (together with these other options), would be better in a separate PR which gives more focus to the features they enable (and maybe adds it to the toolbar?).

Reply:
Yes, I can remove them from this PR and add them when we decide on what we want to show in the markdown preview.